examples/plot_from_raw_to_epochs_to_evoked.py

```diff
@@ -41,15 +41,16 @@
 include = []  # or stim channels ['STI 014']
 raw.info['bads'] += ['EEG 053']  # bads + 1 more
 
-# pick EEG channels
-picks = mne.pick_types(raw.info, meg=False, eeg=True, stim=False, eog=True,
+# pick EEG and MEG channels
+picks = mne.pick_types(raw.info, meg=True, eeg=True, stim=False, eog=True,
                        include=include, exclude='bads')
 # Read epochs
 epochs = mne.Epochs(raw, events, event_id, tmin, tmax, picks=picks,
                     baseline=(None, 0), reject=dict(eeg=80e-6, eog=150e-6),
                     preload=True)
 
-epochs.plot()
+# Plot epochs.
+epochs.plot(trellis=False, block=True, fig_title='Epochs')
```

Comment:
we should maybe find a name that is more intuitive for peeple who haven't studied dataviz and are notoriously using R.